package.json

```diff
@@ -24,17 +25,16 @@
         "husky": "^9.1.6",
         "lint-staged": "^15.2.0",
         "prettier": "^3.3.3",
+        "tsx": "^4.20.3",
```

Comment:
Shouldn't need `tsx` (just tested locally, also need to remove the `--import tsx` from `test`) - I dropped `ts-node` in #62 in favor of built-in type stripping. As long as you use Node.js 24+, it should just work. ✨